src/call_builder.ts

```diff
@@ -149,7 +151,10 @@ export class CallBuilder<
     createEventSource();
     return function close() {
```

Comment:
I noticed before that this function keeps trying to write to `es` even if the assignment fails. I feel like it should return in the catch rather than continuing with the function. Can you make that fix too?

Reply:
Definitely! :)